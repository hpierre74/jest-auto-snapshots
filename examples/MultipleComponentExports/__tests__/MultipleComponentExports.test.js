import snap from 'jest-auto-snapshots';
import { ComponentOne, ComponentTwo } from '../MultipleComponentExports';

snap(ComponentOne, { path: '../MultipleComponentExports.jsx' });
snap(ComponentTwo, { path: '../MultipleComponentExports.jsx' });
