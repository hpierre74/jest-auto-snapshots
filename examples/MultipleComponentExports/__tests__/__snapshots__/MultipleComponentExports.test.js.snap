// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > ComponentOne Matches snapshot when passed only required props: when passed only required props 1`] = `
<span>
  jest-auto-snapshots String Fixture
</span>
`;

exports[`jest-auto-snapshots > ComponentTwo Matches snapshot when passed only required props: when passed only required props 1`] = `
<span>
  1
</span>
`;
